docs/components/vxp-label.md

```diff
@@ -0,0 +1,24 @@
+---
+sidebarDepth: 0
+---
+
+# VxpLabel
```

Comment:
should there be a little explanation what this component does?
There are explanations in other components here

Reply:
@volkanakbayir I think explanation is at line17 it might be moved up here